README.md

```diff
@@ -79,6 +82,9 @@ With output size in mind, we recommend to choose from these theming options:
 To keep your app UI consistent, use the theme properties from `src/lib/theme.scss` also in your
 custom styles.
 
+Icons have been removed from library to reduce its size. Components that require icons allow to pass icon
```

Comment:
I would simply mention that no icons are included in the library and some components accept them via props. `README` is primarily for new-comers and it's fine when the message about removing icons (and why) remains only in change log (release notes).

Reply:
@mbohal wrote that he want to have this information in readme, you two need to decide whether to have it here or not.

Reply:
@mbohal? I'd just say in the README there are no icons included and describe the reason of their removal in change log. What do you think?

Reply:
After phone call with @adamkudrna we clarified that the problem is not the information itself, but the past tense. Users need to know why the icons are not there, they do not need to know why it is so.

I would rephrase it to something like:
` Icons are not included in the library to allow more flexibility and to reduce its size. Components that require icons allow to them to be passed in via props.`

Reply:
Ok, got it.